pyart/aux_io/arm_vpt.py

```diff
@@ -0,0 +1,351 @@
+        'units': 'degrees',
+        'axis': 'radial_elevation_coordinate',
+        'comment': 'Elevation of antenna relative to the horizontal plane',
+    }
+
+    #azimuth = _ncvar_to_dict(ncvars['azimuth'])
+    #elevation = _ncvar_to_dict(ncvars['elevation'])
+
+    if 'scan_rate' in ncvars:
+        scan_rate = cfradial._ncvar_to_dict(ncvars['scan_rate'])
+    else:
+        scan_rate = None
+
+    if 'antenna_transition' in ncvars:
+        antenna_transition = cfradial._ncvar_to_dict(
+            ncvars['antenna_transition'])
+    else:
+        antenna_transition = None
+
+    # 4.9 Moving platform geo-reference variables
+    # Aircraft specific varaibles
+    if 'rotation' in ncvars:
+        rotation = _ncvar_to_dict(ncvars['rotation'])
+    else:
+        rotation = None
+    if 'tilt' in ncvars:
+        tilt = _ncvar_to_dict(ncvars['tilt'])
+    else:
+        tilt = None
+    if 'roll' in ncvars:
+        roll = _ncvar_to_dict(ncvars['roll'])
+    else:
+        roll = None
+    if 'drift' in ncvars:
+        drift = _ncvar_to_dict(ncvars['drift'])
+    else:
+        drift = None
+    if 'heading' in ncvars:
+        heading = _ncvar_to_dict(ncvars['heading'])
+    else:
+        heading = None
+    if 'pitch' in ncvars:
+        pitch = _ncvar_to_dict(ncvars['pitch'])
+    else:
+        pitch = None
+    if 'georefs_applied' in ncvars:
+        georefs_applied = _ncvar_to_dict(ncvars['georefs_applied'])
+    else:
+        georefs_applied = None
+
+    # 4.10 Moments field data variables -> field attribute dictionary
+    if 'ray_n_gates' in ncvars:
+        # all variables with dimensions of n_points are fields.
+        keys = [k for k, v in ncvars.iteritems()
+                if v.dimensions == ('n_points', )]
+    else:
+        # all variables with dimensions of 'time', 'range' are fields
+        keys = [k for k, v in ncvars.iteritems()
+                if v.dimensions == ('time', 'range')]
+
+    fields = {}
+    for key in keys:
+        field_name = filemetadata.get_field_name(key)
+        if field_name is None:
+            if exclude_fields is not None and key in exclude_fields:
+                continue
+            field_name = key
+        fields[field_name] = cfradial._ncvar_to_dict(ncvars[key])
+
+    if 'ray_n_gates' in ncvars:
+        shape = (len(ncvars['time']), len(ncvars['range']))
+        ray_n_gates = ncvars['ray_n_gates'][:]
+        ray_start_index = ncvars['ray_start_index'][:]
+        for dic in fields.values():
+            cfradial._unpack_variable_gate_field_dic(
+                dic, shape, ray_n_gates, ray_start_index)
+
+    # 4.5 instrument_parameters sub-convention -> instrument_parameters dict
+    # this section needed multiple changes and/or additions since the
+    # instrument parameters were primarily located in the global attributes
+    # this section is likely still incomplete
+    omega = float(ncobj.radar_operating_frequency.split()[0])
```

Comment:
Should this be protected with a try or if 'radar_operating_frequency' in ncobj.ncattrs block?